CMakeLists.txt

```diff
@@ -48,82 +48,116 @@ find_package(Threads REQUIRED)
 
 find_package(PkgConfig REQUIRED)
```

Comment:
How does checking for PkgConfig work on your system, but not checking for a PkgConfig package?

Reply:
PkgConfig is required, I didn't change this line.  I only changed the optional dependencies.

Reply:
Yes that is what i was wondering about. If PkgConfig is available, what is the problem of including the headers in the binary build?

Reply:
I think if you include the needed headers with fastfetch it would actually work, because you dlopen the libs anyway.  The only problem was requiring big dependencies to be installed for a header that is only used at build time.